storage/postgres/testdb/testdb.go

```diff
@@ -120,3 +120,13 @@ func sanitize(script string) string {
 func getConnStr(name string) string {
 	return fmt.Sprintf("database=%s %s", name, *pgOpts)
 }
+
+// OpenTestDBOrDie attempts to return a connection to a new postgres
+// test database and fails if unable to do so
```

Comment:
nitty: missing trailing `.`
Actually this is missing throughout, can you pop them on all the comments?